android/engine/src/main/java/org/smartregister/fhircore/engine/util/extension/PatientExtension.kt

```diff
@@ -36,7 +35,7 @@ private fun String.toTitleCase() = replaceFirstChar {
   if (it.isLowerCase()) it.titlecase(Locale.getDefault()) else it.toString()
 }
 
-fun Patient.extractGender(context: Context) =
+fun Patient.extractGender() =
```

Comment:
Removing the context will not allow for translation.

Reply:
context is unused field. I did not find the translation part here